src/components/LoadingComments/LoadingComments.tsx

```diff
@@ -0,0 +1,113 @@
+import React from "react";
+import { css, keyframes } from "emotion";
+
+import { space } from "@guardian/src-foundations";
+import { neutral } from "@guardian/src-foundations/palette";
+
+const BACKGROUND_COLOUR = neutral[93];
+
+const shimmer = keyframes`
+  0% {
+    background-position: -1000px 0;
+  }
+  100% {
+    background-position: 1000px 0;
+  }
+`;
+
+const shimmerStyles = css`
+  animation: ${shimmer} 2s infinite linear;
+  background: linear-gradient(
+    to right,
+    ${BACKGROUND_COLOUR} 4%,
+    ${neutral[86]} 25%,
+    ${BACKGROUND_COLOUR} 36%
+  );
+  background-size: 1000px 100%;
+`;
+
+const containerStyles = css`
+  border-bottom: 1px solid ${neutral[86]};
+  width: 620px;
+  display: flex;
+  padding: ${space[2]}px 0;
+`;
+
+const avatarStyles = (size: number) => css`
+  border-radius: ${size + 10}px;
+  width: ${size}px;
+  height: ${size}px;
+  background-color: ${BACKGROUND_COLOUR};
+  margin-right: ${space[2]}px;
+
+  ${shimmerStyles}
+`;
+
+const Column = ({ children }: { children: JSX.Element | JSX.Element[] }) => (
+  <div
+    className={css`
+      display: flex;
+      flex-direction: column;
```

Comment:
the common use of:
```
display: flex;
flex-direction: column;
```
 and 
```
display: flex;
flex-direction: row;
```
makes me think we should starting thing of some sort of abstraction to avoid duplicate CSS

Reply:
worth making bringing this to src-foundations? as a grid component?

Reply:
Yeah, tempting. The problem is we're not using `styled` so it'a harder to compose components. It'd be great to do: `const PinkRow = styled(Row)'color: pink';` but if we can't then you end up adding lots of divs or creating the propocalypse by adding new props for every new use case 😱